script.js

```diff
@@ -1,57 +1,80 @@
-const collection = [];
+class Library {
+  add() {
+    let existingEntries = JSON.parse(localStorage.getItem('collection'));
+    if (existingEntries == null) existingEntries = [];
+    const entryTitle = document.getElementById('title').value;
+    const entryOwner = document.getElementById('author').value;
+    const book = {
+      title: entryTitle,
+      author: entryOwner,
+    };
+    localStorage.setItem('book', JSON.stringify(book));
+    existingEntries.push(book);
```

Comment:
- [OPTIONAL] I think there is no need for this `localStorage`, I think it would be better to remove it ` localStorage.setItem('book', JSON.stringify(book));` 🤷‍♂️